tests/acceptance/characters/test_operator.py

```diff
@@ -12,6 +13,18 @@
 from tests.utils.middleware import NodeIsDownMiddleware
 
 
+@pytest.fixture(scope="module")
+def vladimir_needs_eth(testerchain):
+    # transfer some ETH to Vladimir who needs it to set provider public key
+    tx = {
+        "to": Vladimir.fraud_address,
+        "from": testerchain.etherbase_account,
+        "value": Web3.to_wei("1", "ether"),
+    }
+    txhash = testerchain.w3.eth.send_transaction(tx)
+    _ = testerchain.wait_for_receipt(txhash)
```

Comment:
I noticed that tx hashes are often assigned to a variable and then ignored (like `_hash` in a different file). Is this some sort of Pythonic idiom?

Reply:
The txhash is used to wait for an tx execution receipt, but the receipt is not needed - it just indicates that the tx was mined/completed.

Some/most of these are removed/replaced in #3227 by using ape functionality for tx execution which does not requiring waiting for a tx receipt - other than the txs for transferring eth to other accounts (haven't gotten around to figuring out the ape equivalent for that as yet, but something I'm looking into for #3227).